src/schema.rs

```diff
@@ -435,3 +436,20 @@ pub struct CTExpectation {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub size: Option<u32>,
 }
+
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+pub struct SynProxy {
+    pub family: NfFamily,
+    pub table: String,
+    pub name: String,
```

Comment:
Should these entries also be documented?

Reply:
Thanks, fixed in 855b72b8dfb9a3010a84c71c290a724e9a75743c

Reply:
I'd prefer to keep the TODO marks in the code base for future code readers to be aware of this.

Created an issue: #70